MyWeatherApp/Models/CityListViewModel.swift

```diff
@@ -18,6 +18,8 @@ class CityListViewModel: ObservableObject {
     func delete(index: IndexSet) {
         cities.remove(atOffsets: index)
     }
+    
+
 }
```

Comment:
Между последней скобкой класса и последней инструкцией тож не ставим пустую строку
```suggestion
    }
}
```